app/src/main/java/grinnell/appdev/edu/lyles/MainActivity.java

```diff
@@ -20,38 +23,57 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        //Set toolbar as the action bar
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
 
         final DrawerLayout drawer = (DrawerLayout) findViewById(R.id.drawer);
 
+        //Enable navigation drawer buttons
         NavigationView nav = (NavigationView) findViewById(R.id.nav_view);
         nav.setNavigationItemSelectedListener(new NavigationView.OnNavigationItemSelectedListener() {
             @Override
             public boolean onNavigationItemSelected(MenuItem item) {
+                //Set "checked" highlight
                 if(item.isChecked()) item.setChecked(false);
                 else item.setChecked(true);
 
+                //Close the drawer when an item is selected
                 drawer.closeDrawers();
 
                 Fragment newFrag;
 
+                //Choose the new fragment based on which item is chosen
                 switch(item.getItemId()) {
                     case R.id.schedule:
                         Log.d("nav", "Schedule selected from nav");
                         newFrag = new ScheduleFragment();
                         break;
+                    case R.id.menu:
+                        Log.d("nav", "Menu selected from nav");
+                        newFrag = new MenuFragment();
+                        break;
+                    case R.id.beer:
+                        Log.d("nav", "Beer selected from nav");
+                        newFrag = new BeerFragment();
+                        break;
+                    case R.id.favorites:
+                        Log.d("nav", "Favorites selected from nav");
```

Comment:
I also like that you're constantly logging debug statements to make testing and observations easy!